samples/42.scaleout/src/main/java/com/microsoft/bot/sample/scaleout/Application.java

```diff
@@ -58,7 +58,12 @@ public Bot getBot() {
         // String accountName = "<ACCOUNT-NAME>";
         // String accountKey = "<ACCOUNT-KEY>";
         // String container = "dialogs";
-        // Store store = new BlobsStore(accountName, accountKey, container);
+        // Store store = null;
+        // try {
+        //     store = new BlobStore(accountName, accountKey, container);
+        // } catch (URISyntaxException | StorageException e) {
+        //     e.printStackTrace();
+        // }
```

Comment:
I think that this can be avoided if we handle the exception in the constructor instead of the initialization